api/plone-5.2.x.cfg

```diff
@@ -8,3 +8,4 @@ versions=versions
 
 [versions]
 plone.restapi =
+lxml = 4.4.2
```

Comment:
Please do not mix things in PRs. If this is needed because some other reason, please create a PR for it.